client/src/app/data/backup-quotes.json

```
@@ -1,8002 +1,8002 @@
 [
   {
-    "date": "2016-01-05T00:00:00.000Z",
+    "date": "2016-01-04T00:00:00.000Z",
```

Use dates without timezone awareness

```suggestion
    "date": "2016-01-04",
```